base/setup/usetup/format.c

```diff
@@ -104,6 +104,12 @@ FormatPartition(
 
     ProgressSetStepCount(FormatProgressBar, 100);
 
+    /*
```

Comment:
For consistency with the rest of the code, this MUIClearStyledText call should be done instead there:
https://git.reactos.org/?p=reactos.git;a=blob;f=base/setup/usetup/usetup.c;hb=6ad4f9e7bbab0effbdb18423f842dc6ab1d1dc12#l3389
within the handling of the ENTER key: you can place this call just before the call that sets the status text to STRING_PLEASEWAIT.